### ISSUE TITLE
Certain default themes in PySimpleGUI break the 'reskin' command.

### ISSUE THREAD

**Issue by richnanney** [definite-d/PSG_Reskinner#7]
Good afternoon,

I stumbled on your project from the comments on an issue raised in the PSG repository. I wanted to add a 'randomize theme' button in my project and your library serves my needs perfectly.

I found, however, that certain default themes found in PSG break the 'reskin' command. I found this from using the following script:

```python
import PySimpleGUI as sg
from random import *
import psg_reskinner as psgr

def randomize_theme():
    new_theme = sg.theme_list()[randrange(0, 154)]
    while new_theme in [
        "SystemDefaultForReal",
        "Default1",
        "Default",
        "SystemDefault",
        "DefaultNoMoreNagging",
        "SystemDefault1",
        "GrayGrayGray",
    ]:
        new_theme = sg.theme_list()[randrange(0, 154)]
    return new_theme

layout = [[sg.T("Hello world!")], [sg.B("Randomize Theme"), sg.B("Exit")]]

window = sg.Window("Random Themes", layout=layout)
while True:
    event, values = window.read(300)

    if event == sg.WIN_CLOSED or event == "Exit":
        break

    if event == "Randomize Theme":
        new_theme = randomize_theme()
        psgr.reskin(
            window=window,
            new_theme=new_theme,
            theme_function=sg.theme,
            lf_table=sg.LOOK_AND_FEEL_TABLE,
        )
```

The PSG theme list has 154 different themes, and I went through and wrote down each default theme that gives an error, a list of which is in the previous code block. 

The error that I would get is as follows:

```
Traceback (most recent call last):
  File "c:\...\theme_testing.py", line 31, in <module>
    psgr.reskin(
  File "C:\...\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\psg_reskinner\__main__.py", line 395, in reskin
    window.TKroot.configure(background=new_theme_dict['BACKGROUND'])
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.11_3.11.1264.0_x64__qbz5n2kfra8p0\Lib\tkinter\__init__.py", line 1702, in configure
    return self._configure('configure', cnf, kw)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.11_3.11.1264.0_x64__qbz5n2kfra8p0\Lib\tkinter\__init__.py", line 1692, in _configure
    self.tk.call(_flatten((self._w, cmd)) + self._options(cnf))
_tkinter.TclError: unknown color name "1234567890"
```

I recognize, of course, that this was probably a smaller project and I would hate to dredge up an annoyance for you -- that is to say that this issue is easily worked around for the user. Just thought I would bring it to your attention. 
Thanks, and have a pleasant day.

**Comment by richnanney** [comment]
Also, within my testing, I've found that TabGroups and Tabs also break the command. I can open up a new issue, if you like.

**Comment by definite-d** [comment]
Thanks for your interest in the project, @richnanney ! To be honest, I've been aware of this error, and the cause requires a bit of an explanation to fully understand, so:


TLDR: 
_The best solution to the issue (for now) is to either avoid themes with `COLOR_SYSTEM_DEFAULT` (or `1234567890`) in their values, or create a custom theme that contains the actual color values for the desired default-look theme, and switch to that instead._


Long Form Explanation:

```
_tkinter.TclError: unknown color name "1234567890"
```

The error you encounter with that `1234567890` number is fundamentally a PySimpleGUI thing, specifically the `COLOR_SYSTEM_DEFAULT` variable ; to quote the source code, it says the variable is...

> `# A Magic Number kind of signal to PySimpleGUI that the color should not be set at all`

... and as you could probably tell, the value of that variable is 1234567890, and it's used about 260 times throughout PySimpleGUI's code, including default themes. 

This is because the default approach to theming in PySimpleGUI is to set only certain colors and ignore the rest, leaving Tkinter to decide, and PySimpleGUI uses that number to signify color values to ignore.

While Reskinner could follow the same route and simply ignore changes if `COLOR_SYSTEM_DEFAULT` is detected, it's not a good solution, primarily because it wouldn't update whatever color was set previously. A good solution would be to substitute a color internally (within Reskinner) by hand, but different OSes have different default colors for specific elements, which only Tkinter knows how to access.

Hence, the best solution to the issue (for now) is to either avoid themes with `COLOR_SYSTEM_DEFAULT` (or `1234567890`) in their values, or create a custom theme that contains the actual color values for the desired default-look theme, and switch to that instead. I'll work on dynamically obtaining the default colors, so this issue can be resolved once and for all.


Once again, thank you for using the project. Your feedback is appreciated.

**Comment by definite-d** [comment]
> Also, within my testing, I've found that TabGroups and Tabs also break the command. I can open up a new issue, if you like.

Regarding this, yes, I would appreciate it if you opened a new issue for that.

**Comment by definite-d** [comment]
Hello again, @richnanney .

I've released v2.3.13 on PyPi, which contains a solution that should (in the meantime) solve this issue to a good degree. A simple `python -m pip install --upgrade PSG-Reskinner` should do. 

Please try it out and let me know if it satisfies your needs.

**Comment by richnanney** [comment]
Looks like everything works -- and wonderfully. Thanks for your work!
